ElementX/Sources/Screens/Settings/DeveloperOptionsScreen/View/DeveloperOptionsScreen.swift

```diff
@@ -51,6 +51,13 @@ struct DeveloperOptionsScreen: View {
                 }
             }
             
+            Section("Knocking") {
+                Toggle(isOn: $context.knockingEnabled) {
+                    Text("Allow the usage of the knock join rule")
```

Comment:
Optional, just thinking this might be a bit more succinct:
```suggestion
            Section("Join rules") {
                Toggle(isOn: $context.knockingEnabled) {
                    Text("Knocking")
```